### DISCUSSION TITLE
React 18

### DISCUSSION THREAD

**Discussion by MincePie** [chakra-ui/chakra-ui#6874] [Q&A] [answered]
I am trying to figure out how to use the Chakra UI pricing page with React 18 in my app.

I am currently getting an error that says:

_Error: Objects are not valid as a React child (found: object with keys {name, description, price, features}). If you meant to render a collection of children, use an array instead._

The error points to this page. I don't have an object defined with the keys noted in the error message so am confused about how to find a solution. I have tried putting fragment keys around various parts of this code, but can't find a formulation that works to address the error.

Do you know what change needs to be made to the code to get it to work with react 18?

[import { Box, Container, Heading, Stack, Text, useBreakpointValue } from '@chakra-ui/react'
import * as React from 'react'
import { products } from '../../components/footer/links/pricing/data'
import { ProductCallToActionTable } from '../../components/footer/links/pricing/ProductCallToActionTable'
import { ProductFeaturesTable } from '../../components/footer/links/pricing/ProductFeaturesTable'
import { ProductOverviewTable } from '../../components/footer/links/pricing/ProductOverviewTable'
import { HomeLayout } from "components/HomeLayout"

export default function Pricing() {

  const isMobile = useBreakpointValue({ base: true, md: false })
  return (
    <>
    
    <Box as="section" bg="bg-surface" maxWidth="100%" >
      <Container py={{ base: '16', md: '24' }}  maxWidth="100%">
        <Box as="section" bg="bg-surface" py={{ base: '16', md: '24' }}>
        <Stack spacing={{ base: '16', md: '24' }} shouldWrapChildren>
            <Container maxW="100%">
                <Stack spacing={{ base: '4', md: '6' }} textAlign="center" align="center">
                    <Stack spacing="3">
                   
                    <Heading size={useBreakpointValue({ base: 'md', md: 'lg' })} color="brand.orange">
                        Offer
                    </Heading>
                    </Stack>
                    <Text fontSize={{ base: 'lg', md: 'xl' }} color="muted" maxW="100%">
                    Get access.
                    </Text>
                </Stack>
            </Container>
            {isMobile ? (
            <Stack spacing="16">
                {products.map((product, id) => (
                <Container px={{ base: '0' }} key={id} maxW="100%">
                    <Stack spacing={{ base: '5', md: '7' }}>
                    <ProductOverviewTable products={[product]} />
                    <ProductFeaturesTable products={[product]} />
                    <ProductCallToActionTable products={[product]} />
                    </Stack>
                </Container>
                ))}
            </Stack>
            ) : (
            <Container px={{ base: '0', lg: '8' }} maxW="80%" ml="10%">
                <Stack spacing={{ base: '5', md: '7' }}>
                <ProductOverviewTable products={products} />
                <ProductFeaturesTable products={products} />
                <ProductCallToActionTable products={products} />
                </Stack>
            </Container>
            )}
        </Stack>
        </Box>
    
        </Container>
    </Box>
    
    </>
  )
}

Pricing.getLayout = (page: React.ReactNode) => <HomeLayout>{page}</HomeLayout>](url)

**Comment by thiswallz** [comment, marked as answer]
I think is a children component, try to search ProductCallToActionTable, ProductFeaturesTable, ProductOverviewTable and others
and look if you are trying to print something like
```
<div>
{ object }
</div>
```

where object can be whatever you have, and I think you missed to use { object.name } fro instance, since this error is common when you are trying to print an object without referencing the attribute.
